ACadSharp/Objects/MultiLeaderAnnotContextClasses.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.Collections.Generic;
+
+using ACadSharp.Attributes;
+using ACadSharp.Tables;
+
+using CSMath;
+
+namespace ACadSharp.Objects
+{
+
+	/// <summary>
+	/// Common AcDbAnnotScaleObjectContextData data (see paragraph 20.4.71).
+	/// 300 DXF: “CONTEXT_DATA{“
+	/// </summary>
+	public partial class MultiLeaderAnnotContext : CadObject
```

Comment:
The documentation for this class should be revised, I see that is copied from the PDF.